ssm/ssm/projects/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.contrib import admin
+from model_utils.choices import Choices
+
+from ssm.core.models import BaseModel
+from ssm.users.models import User
+
+STATUS = Choices(('started', 'Started'), ('in progress', 'In progress'), ('completed', 'Completed'))
+ROLES = Choices(('project manager', 'Project manager'), ('developer', 'Developer'), ('tester', 'Tester'),
+                ('business analyst', 'Business analyst'), ('front-end developer', 'Front-end developer'),
+                ('back-end developer', 'Back-end developer'), ('data scientist', 'Data scientist'),
+                ('designer', 'Designer'), ('team lead', 'Team lead'), ('product owner', 'Product owner'))
+
+
+class Project(BaseModel):
+    name = models.CharField('Name', max_length=32, unique=True)
```

Comment:
It doesn't guarantee "true" uniqueness here as "Project" and "pRoject" values will not be recognized as one value, also you don't handle spaces at the beginning or at the end of name, so "Project" and "Project " also will be different names, you will need to design uniqueness rule here, for example use `[title()](https://docs.python.org/3/library/stdtypes.html#str.title)` and `[strip()](https://docs.python.org/3/library/stdtypes.html#str.strip)` combination () for `name` value, the same problem was for `Skill` model where `save(...)` method was modified to always force `lower()` and `strip()`: 
```
    def save(self, *args, **kwargs):
        self.name = self.name.lower().strip()
        super().save(*args, **kwargs)
```